src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="product-image" 
+        class="product-card__product-image"
+      >
+
+      <div class="product-card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="product-rating product-card__product-rating">
+        <div class="product-rating__stars">
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+          <div class="product-rating__star"></div>
+        </div>
+
+        <div class="product-rating__reviews">
+          Reviews: 5
+        </div>
+      </div>
```

Comment:
```suggestion
      <div class="card__rating">
        <div class="card__stars stars">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <div class="card__reviews">
          Reviews: 5
        </div>
      </div>
```
Product rating is not a separate block, just element of card.
While stars is a block from previous task. 
You can read more about BEM methodology here:
https://css-tricks.com/bem-101/
https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/

Reply:
Done